policy/modules/kernel/domain.if

```diff
@@ -76,6 +76,8 @@ interface(`domain_type',`
 	# start with basic domain
 	domain_base_type($1)
 
+	kernel_read_crypto_sysctls($1)
```

Comment:
This should be in domain.te an work on the `domain` attribute directly.

Reply:
Yes, I should have realized that initially.